[Downloader] `[p]findcog` doesn't work if cog class is at different level than `cog_folder/somefile.py`

# Command bugs

#### Command name

`[p]findcog`

#### What cog is this command from?

Downloader

#### What were you expecting to happen?

Get cog the command is from

#### What actually happened?

Got error:
```
[2019-12-08 23:17:40] [ERROR] red: Exception in command 'findcog'
Traceback (most recent call last):
  File "venv\lib\site-packages\discord\ext\commands\core.py", line 79, in wrapped
    ret = await coro(*args, **kwargs)
  File "venv\lib\site-packages\redbot\cogs\downloader\downloader.py", line 1199, in findcog
    cog_name = self.cog_name_from_instance(cog)
  File "venv\lib\site-packages\redbot\cogs\downloader\downloader.py", line 1182, in cog_name_from_instance
    return splitted[-2]
IndexError: list index out of range
```

#### How can we reproduce this issue?

1. Create a cog with cog class placed in `__init__.py`
2. Load the cog.
3. Use `[p]findcog command_from_that_cog`
4. See error

---

In short, it looks like `Downloader.cog_name_from_instance` heavily relies on most common file structure in cogs where you have cog class in `cog_folder/somefile.py` file and it raises if it's in `cog_folder/__init__.py` and also determines wrong cog name if it's placed in `cog_folder/subpackage/somefile.py`

--- a/changelog.d/downloader/3177.bugfix.rst
+++ b/changelog.d/downloader/3177.bugfix.rst
@@ -1,0 +1,1 @@
+``[p]findcog`` now properly works for cogs with less typical folder structure.

--- a/redbot/cogs/downloader/downloader.py
+++ b/redbot/cogs/downloader/downloader.py
@@ -1179,7 +1179,7 @@ class Downloader(commands.Cog):
 
         """
         splitted = instance.__module__.split(".")
-        return splitted[-2]
+        return splitted[0]
 
     @commands.command()
     async def findcog(self, ctx: commands.Context, command_name: str) -> None:

